### PR TITLE
feat: configure tooltip position in Grid

### PR DESCRIPTION
**Pull request by sissbruecker**
Adds a `setTooltipPosition` API to configure the position of tooltips in the grid. For now this applies for tooltips on all cells, it would not be possible to configure individual positions per column.

Closes https://github.com/vaadin/flow-components/issues/6974

### CODE REVIEW

File: vaadin-grid-flow-parent/vaadin-grid-flow/src/main/java/com/vaadin/flow/component/grid/Grid.java [hunk 1]
```diff
@@ -4804,9 +4805,22 @@ public void setTooltipGenerator(
...
     }
 
+    /**
+     * Sets the tooltip position relative to the cell that is being hovered or
+     * focused. The default position is {@link TooltipPosition#BOTTOM}.
+     *
+     * @param position
+     *            the position to set
+     */
+    public void setTooltipPosition(TooltipPosition position) {
```
Comment (vursen) [inline vaadin-grid-flow-parent/vaadin-grid-flow/src/main/java/com/vaadin/flow/component/grid/Grid.java:4815]: Would it make sense to add a getter for consistency?

Comment (vursen) [inline vaadin-grid-flow-parent/vaadin-grid-flow/src/main/java/com/vaadin/flow/component/grid/Grid.java:4815]: Setting `null` currently results in `java.lang.NullPointerException`.

Comment (sissbruecker) [reply vaadin-grid-flow-parent/vaadin-grid-flow/src/main/java/com/vaadin/flow/component/grid/Grid.java:4828]: Added a getter. To facilitate this, added a method to the enum to get an enum value from the position string, which returns the bottom position by default.

Comment (sissbruecker) [reply vaadin-grid-flow-parent/vaadin-grid-flow/src/main/java/com/vaadin/flow/component/grid/Grid.java:4828]: Done

**Review by vursen — APPROVED** [review]

**Pull request merged**